spec/services/recurring_contribution/subscriptions/create_juntos_spec.rb

```diff
@@ -0,0 +1,97 @@
+require 'rails_helper'
+
+RSpec.describe RecurringContribution::Subscriptions::CreateJuntos do
+  describe ".process" do
+    let(:project)     { create(:project) }
+    let(:plan)        { create(:plan) }
+
+    context "when all parameters sent are valid" do
+      let(:service_response) do
+        RecurringContribution::Subscriptions::CreateJuntos.process(
+          project:          project,
+          plan:             plan,
+          user:             project.user,
+          payment_method:   'bank_billet',
+          charging_day:     5
+        )
+      end
+
+      context "when the payment_method is credit card" do
+        let(:credit_card_attributes) { build(:credit_card) }
+        let(:pagarme_response_for_card_creation) { double('Card', id: 'card_ci6y37h16wrxsmzyi') }
+        let(:service_response) do
+          RecurringContribution::Subscriptions::CreateJuntos.process(
+            project:          project,
+            plan:             plan,
+            user:             project.user,
+            payment_method:   'credit_card',
+            charging_day:     5,
+            credit_card_hash: credit_card_attributes
+          )
+        end
+
+        it "should create a PagarMe::Card" do
+          allow(Pagarme::API).to receive(:create_credit_card)
+            .with(credit_card_attributes).and_return(pagarme_response_for_card_creation)
+
+          expect(service_response.credit_card_key).to eq 'card_ci6y37h16wrxsmzyi'
+        end
+      end
+
+      it "should create a subscription" do
+        expect(service_response)
+          .to have_attributes(status: :waiting_for_charging_day, plan_id: plan.id, credit_card_key: nil)
```

Comment:
Aqui tá faltando você ver se essa subscription que você criou de fato está no banco. Nesse caso deveria ser algo como "should return the subscription" e então você adicionaria um outro spec pra "should create a subscription" e verificaria esse esquema do banco :)